keyboards/s_ol/0xc_pad/readme.md

```diff
@@ -0,0 +1,24 @@
+# 0xC.pad
+
+![0xC.pad](https://i.imgur.com/YXMKy4u.png)
+
+A 12-key, RGB-backlit hexagonal macropad.
+
+* Keyboard Maintainer: [s-ol](https://github.com/s-ol)
+* Hardware Supported: "hex12pad r0", 0xC.pad rev1
+* Hardware Availability: [IC form](https://s-ol.nu/0xC.pad)
+
+To flash the example keymap for this macropad
+(after setting up your build environment):
+​
+    make s_ol/0xc_pad:default:flash
+
+Before flashing, the macropad must be brought into DFU mode by one of the
+following means:
+- holding down the top-left key (next to the USB port) while plugging it in
+- bridging the two pads labeled `BOOT` on the back of the PCB with a metallic
+  object (tweezers, paperclip. wire)
+- putting the `RESET` keycode in your keymap and pressing the corresponding key
+​
+See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```

Comment:
Workaround for a thing with GitHub's Markdown parser, which gets weird around lists.

![image](https://user-images.githubusercontent.com/18669334/110156025-4ff82f00-7d9b-11eb-8599-15f8ac9fd504.png)

```suggestion

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information.
```